projects/angular/program-website/src/app/routes/home/home.component.html

```diff
@@ -1,155 +1,134 @@
+<nav>
+  <div class="logo">OSMP</div>
+  <div class="menu">
+    <a [routerLink]="'profile-pages/'">PROJECTS</a>
+    <a href="https://www.hasadna.org.il/participants/">PARTICIPANTS</a>
+    <a href="https://www.hasadna.org.il/">ABOUT</a>
+    <a href="https://www.hasadna.org.il/contact/">CONTACT US</a>
+  </div>
+</nav>
 <div class="program-website-container">
-  <div class="top-section">
-    <div class="image-1-container">
-      <img src="assets/images/image-1.svg" class="image-1">
-    </div>
-    <div class="navigation-bar">
-      <nav class="navigation">
-        <a [routerLink]="'profile-pages/'">PROJECTS</a>
-        <a href="https://www.hasadna.org.il/participants/">PARTICIPANTS</a>
-        <a href="https://www.hasadna.org.il/">ABOUT</a>
-        <a href="https://www.hasadna.org.il/contact/">CONTACT US</a>
-      </nav>
+  <div class="section section-header">
+    <div class="responsive">
+      <img src="assets/images/image-1.svg">
     </div>
-    <div class="osmp-container">
-      <p class="osmp-title">
-        Open Source Mentorship Program
-      </p>
-      <p class="osmp-text">
+    <div class="content">
+      <div class="osmp-title">Open Source Mentorship Program</div>
+      <div class="osmp-text">
         Lorem ipsum dolor sit amet, consectetur adipiscing elit,
         sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
         Venenatis a condimentum vitae sapien pellentesque habitant morbi tristique senectus.
-      </p>
+      </div>
       <div class="get-started-button">
         <a class="get-started-link" href="https://www.hasadna.org.il/get-started/">
           GET STARTED
         </a>
       </div>
     </div>
   </div>
-  <div class="about-section">
-    <div class="about-container">
-      <p class="about-title">
+  <div class="section section-about">
+    <div class="content">
+      <div class="about-title">
```

Comment:
I think `<p>` is not the best tag to set basic structure, because it contains hidden style inside.
It's more suitable to separate text to paragraphs.

Reply:
Got it.